test/app/invitations_test.rb

```diff
@@ -89,6 +89,14 @@ def test_invitation
     refute team.includes?(charlie)
   end
 
+  def test_team_creator_receive_no_notification
```

Comment:
Sorry, I think I'm being confusing... I meant like basically the same test about `test_user_must_be_manager` but adding both alice and bob.

Something like:
```ruby
team = Team.by(alice).defined_with(slug: 'abc', usernames: bob.username)
team.save

verb, path, action = [:post, '/teams/abc/invitations', "invite members"]
location = "http://example.org/teams/abc"

send verb, path, { usernames: "#{charlie.username}, #{alice.username}" }, login(alice)

assert_equal 1, bob.reload.team_membership_invites.count
assert_equal 0, alice.reload.team_membership_invites.count
```

See I think there's a bug where we are using `params[:users]` and it's not defined. So that code path to remove the inviter from the list of invitees doesn't run.

I just ran this on my machine and got an invite as alice.